src/main/kotlin/no/nav/pensjon/pdfmerger/advancedMerge/AdvancedPdfMerger.kt

```diff
@@ -0,0 +1,211 @@
+package no.nav.pensjon.pdfmerger.advancedMerge
+
+import com.lowagie.text.*
+import com.lowagie.text.pdf.*
+import no.nav.pensjon.pdfmerger.Dokumentinfo
+import no.nav.pensjon.pdfmerger.MergeInfo
+import java.io.ByteArrayOutputStream
+import java.time.format.DateTimeFormatter
+import kotlin.collections.ArrayList
+
+class AdvancedPdfMerger(
```

Comment:
Jeg syns at denne klassen gjør for mye. Den merker både PDF-dokumenter og lager et skilleark, forside ark eller noe sånt. Dette burde være minst to klasser. Skilleark og merger. Ta frem "separation of concerns" hatten og se streng på koden, da tipper jeg resultatet blir meget bra.